server/package.json

```diff
@@ -5,16 +5,19 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "start": "node index.ts",
+    "start": "npm run serve",
```

Comment:
"start" is merely an alias to "serve" - why not use `"start": "node ../dist/index.js"` instead?